capty.py

```
@@ -253,104 +950,152 @@ def on_record_clicked(self, btn):
+            logger.info(f"Stopping recording process (PID: {self.ffproc.pid})")
             os.killpg(os.getpgid(self.ffproc.pid), signal.SIGINT)
             self.status.set_text("Stopping...")
+            # Reset state after stopping
+            self.reset_recording_state()
         except Exception as e:
-            print("Stop error:", e)
+            logger.error(f"Stop error: {e}")
             try:
                 self.ffproc.terminate()
+                logger.info("Process terminated")
             except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)